### ISSUE TITLE
fix python 2.7 image

### ISSUE THREAD

**Issue by dminca** [skullops/dockerfiles#11]
```sh
➜  ~  docker run --rm -ti wizdevops/python:2.7-alpine python          
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"python\\\": executable file not found in $PATH\"\n".
```

**Comment by dminca** [comment]
Image works perfectly, just a flaw in build process (images weren't being build because `force: true` wasn't used in ansible task)

``` sh

➜  open-source/dockerfiles git:(master) docker run --rm -ti wizdevops/python:2.7-alpine python --version
Python 2.7.12
```

**Comment by dminca** [comment]
Closed by 24506f8
